ompi/mca/osc/ucx/osc_ucx_comm.c

```diff
@@ -358,7 +358,9 @@ static inline int get_dynamic_win_info(uint64_t remote_addr,
     free(temp_buf);
 
     /* unlock the dynamic lock */
-    return ompi_osc_ucx_dynamic_unlock(module, target);
+    ret_unlock = ompi_osc_ucx_dynamic_unlock(module, target);
+    /* ignore unlock result in case of error */
+    return (OPAL_SUCCESS != ret) ? ret : ret_unlock;
```

Comment:
Mess up with error codes, this function should not return OPAL_SUCCESS but OMPI_SUCCESS. See https://github.com/open-mpi/ompi/blob/0edb3114b6afea1c14df13f064ce6e9a9dbb19a7/ompi/mca/osc/ucx/osc_ucx_comm.c#L30-L36

Reply:
See #11030 for a cleanup of the returned error code types.

Reply:
yep, fixed